src/components/BookContainer.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useSelector } from 'react-redux/es/exports';
+import AddBook from './AddBook';
+
+const BookContainer = () => {
+  const books = useSelector((state) => state.booksReducer);
+
+  return (
+    <div className="main-container">
+      {books.map((book) => (
+        <div key={book.id}>
+          <li className="books-content">
+            {book.title}
+            {' by '}
+            {book.author}
+            {' - '}
+            {book.category}
+          </li>
+          <button type="button" id={book.id}>Remove</button>
+        </div>
```

Comment:
- [ ] Kindly, make sure that your `Remove` button to delete a book works as expected. For now, when I click on it, nothing happens. Make sure that you created a function for removing a book and you pass it as the event handler. means you'll use the `onClick{pass your function}`

Reply:
Hi @RolandM99! The project requirement  is  that the remove button is suppose to work in the next project not this on. when adding selectors and dispatch.